### DISCUSSION TITLE
Is Raspberry Pi 3B+ Supported?

### DISCUSSION THREAD

**Discussion by SamuelBanya** [retronas/discussions#120] [General]
Hey there,

I have a question, does anyone know if a Raspberry Pi 3B+ supported?

Asking since I have a spare one laying around that I could use for this but wasn't sure if this would be ideal, or exactly powerful enough for this.

I did check the related supported hardware section:
https://github.com/retronas/retronas/wiki/Supported-hardware

But I did have a friend tell me that a 3B+ probably isn't powerful enough for this so I wanted to confirm, thanks.

**Comment by sairuk** [comment]
the retronas scripts will run on a 3b+ with linux, but you need to understand retronas is a collection of installers and preconfigured 3rd party tools so when asking if something is supported it depends on what you choose to install and run on the hw

it may be under powered for some things so we suggest an rpi4. the rpi3 is perfectly capable of running a samba service for example. The install will take longer than an rpi4

I have a retronas installed/configured iscsi usb gadget setup on an rpi zero w, the installer was slow but it works fine. I wouldn't run multiple tools off it.

retronas is a playground, go try things and feedback to the community
